UnityProject/Assets/Scripts/Objects/FlasherBase.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections;
+using AddressableReferences;
+using Mirror;
+using NaughtyAttributes;
+using Player;
+using UnityEngine;
+
+namespace Objects
+{
+	public class FlasherBase : NetworkBehaviour
```

Comment:
why does it need to be NetworkBehaviour?

Reply:
Interactions and because the flash stuff only works on the server (take a look at the stun for example)